src/sql/workbench/browser/modelComponents/groupContainer.component.ts

```diff
@@ -14,11 +14,14 @@ import { GroupLayout, GroupContainerProperties } from 'azdata';
 
 import { ContainerBase } from 'sql/workbench/browser/modelComponents/componentBase';
 import { endsWith } from 'vs/base/common/strings';
+import { StandardKeyboardEvent } from 'vs/base/browser/keyboardEvent';
+import { KeyCode } from 'vs/base/common/keyCodes';
+import * as DOM from 'vs/base/browser/dom';
 
 @Component({
 	selector: 'modelview-groupContainer',
 	template: `
-		<div *ngIf="hasHeader()" [class]="getHeaderClass()" (click)="changeState()">
+		<div *ngIf="hasHeader()" [class]="getHeaderClass()" (click)="changeState()" [tabindex]="isCollapsible()? 0 : -1" [attr.role]="isCollapsible() ? 'button' : null" [attr.aria-expanded]="isCollapsible() ? !collapsed : null">
```

Comment:
I assume you verified null works as expected here? For Element.setAttribute it warns that null would be converted to the string null which wouldn't be what we want : https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute

(I don't know what angular is using for setting the value though). Regardless I've always just used an empty string instead for clearing a value - that's a closer representation of what we actually want to happen anyways so I generally prefer that.

Reply:
yes, tested. here are the screenshots.

collapsable:
<img width="719" alt="Screen Shot 2019-11-24 at 12 17 57 PM" src="https://user-images.githubusercontent.com/13777222/69500955-e3fafb80-0eb4-11ea-8e94-5bd59b573968.png">

![Screen Shot 2019-11-24 at 12 17 42 PM](https://user-images.githubusercontent.com/13777222/69500957-f2e1ae00-0eb4-11ea-898a-77b0b522ff32.png)

non-collapsable:

![Screen Shot 2019-11-24 at 12 19 56 PM](https://user-images.githubusercontent.com/13777222/69500966-08ef6e80-0eb5-11ea-8f45-28b998bac344.png)
<img width="691" alt="Screen Shot 2019-11-24 at 12 20 03 PM" src="https://user-images.githubusercontent.com/13777222/69500967-08ef6e80-0eb5-11ea-9055-8b1ee27be994.png">